### PR TITLE
New litany: Search

### PR DESCRIPTION
**Pull request by TheLeopold**
## About The Pull Request
_(yay, another NT buff from me, who could have expected)_
Adds knock-off of inquisitor's Scrying litany for finding lost people: Search. All it does is tells the user the location of selected disciple. (similar to Entreaty)
Can be used by acolytes. Requires facing EOTP. Costs 25 energy. So it can't be used to reliably track murderer or someone hiding. It's also different from suit sensors or that removed Eye miracle, in that you have to be already looking for said person.
Target doesn't receive any notification, because 1984.

## Why It's Good For The Game
Gives NT real way of finding dead disciples, without giving suit sensors console to church or disabling the counterplay (keep low profile/move, remove cruciform/throw body in space)

## Testing
used litany, used litany facing EOTP, selected disciple.
I wasn't able test if it works with dead disciples, as I believe dead people needs ckey to be displayed. 

## Changelog
:cl:
add: Added new acolyte's litany: Search
/:cl:

### CODE REVIEW

**Comment by DiaFRAME444** [comment]
A feature similar to this was requested in Kotmap (Although the exact implementation is different), so its something NT was always meant to have.
The 50 energy cost seems too high.

**Comment by MLGTASTICa** [comment]
> A feature similar to this was requested in Kotmap (Although the exact implementation is different), so its something NT was always meant to have. The 50 energy cost seems too high.

get your holy life up heretic

**Comment by assortedbeads** [comment]
50 enemy is lame, make it like 25

File: code/modules/core_implant/cruciform/rituals/priest.dm [hunk 1]
```diff
@@ -508,3 +508,26 @@
...
+	var/obj/machinery/power/eotp/EOTP = locate(/obj/machinery/power/eotp) in OBJS
+	if(!EOTP)
+		fail("You must be in front of the Eye of the Protector.", user, C)
+		return FALSE
+
+	var/mob/living/carbon/human/H = pick_disciple_global(user, TRUE)
+	if (!H)
+		return
+	var/area/t = get_area(H)
+	log_and_message_admins("finds location of [H] with search litany")
```
Comment (TheShown911) [inline code/modules/core_implant/cruciform/rituals/priest.dm:531]: Does this need to be logged and messaged to admins?

File: code/modules/core_implant/cruciform/rituals/priest.dm [hunk 2]
```diff
@@ -508,3 +508,26 @@
...
+	to_chat(user, "<span class='info'>[H], faithful cruciform follower, currently at [t.name].</span>")
```
Comment (TheShown911) [inline code/modules/core_implant/cruciform/rituals/priest.dm:532]: Use span macros here.

**Review by hyperioo — CHANGES_REQUESTED** [review]
See above

File: code/modules/core_implant/cruciform/rituals/priest.dm (hunk 1 above)
Comment (TheLeopold) [reply code/modules/core_implant/cruciform/rituals/priest.dm:531]: I don't know, I copied it from Entreaty. You tell me.

File: code/modules/core_implant/cruciform/rituals/priest.dm (hunk 2 above)
Comment (TheLeopold) [reply code/modules/core_implant/cruciform/rituals/priest.dm:532]: what?

File: code/modules/core_implant/cruciform/rituals/priest.dm (hunk 1 above)
Comment (hyperioo) [reply code/modules/core_implant/cruciform/rituals/priest.dm:531]: Use a `log_game` instead and add the location in the log ([key_name(user)] locates [H] in [t] using Search litany)

**Review by hyperioo — APPROVED** [review]

**Pull request merged**